File: week1/pset1/mario.c
#include <stdio.h>
#include <cs50.h>

void printmario(int n);

int main(void)
{
    int n;
    while(true){
        n = get_int("Height: ");
        if(n >=0 && n <= 23){
            break;
        }
    }
    printmario(n);
}


void printmario(int n)
{
    for(int i=0; i<n; i++){
        for(int j=0; j<n-i-1; j++){
            printf(" ");
        }
        for(int j=0; j<i+1; j++){
            printf("#");
        }
        printf("  ");
        for(int j=0; j<i+1; j++){
            printf("#");
        }
        printf("\n");
    }
}

File: lecture9/pset6/mario.py
from cs50 import get_int

def printmario(n):
    for i in range(n):
        s = ""
        for j in range(n-i-1):
            s += " "
        for j in range(i+1):
            s += "#"
        s += "  "
        for j in range(i+1):
            s += "#"
        print(s)

while(True):
    n = get_int("Height: ")
    if (n >= 0 and n <= 23):
        break
printmario(n)

